.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "xo.enable": true
```

Comment:
No editor specific config files please. `.vscode` can go in your global gitignore.